app/code/Magento/Checkout/Block/Cart/Shipping.php

```diff
@@ -3,54 +3,73 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+
 namespace Magento\Checkout\Block\Cart;
 
+use Magento\Checkout\Model\CompositeConfigProvider;
+use Magento\Checkout\Block\Checkout\LayoutProcessorInterface;
+use Magento\Framework\Serialize\Serializer\Json;
+use Magento\Framework\Serialize\Serializer\JsonHexTag;
+use Magento\Framework\View\Element\Template\Context;
+use Magento\Customer\Model\Session as customerSession;
+use Magento\Checkout\Model\Session as checkoutSession;
+use Magento\Framework\App\ObjectManager;
+
 /**
+ * Cart Shipping Block
+ *
  * @api
  * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
  */
 class Shipping extends \Magento\Checkout\Block\Cart\AbstractCart
 {
     /**
-     * @var \Magento\Checkout\Model\CompositeConfigProvider
+     * @var CompositeConfigProvider
      */
     protected $configProvider;
 
     /**
-     * @var array|\Magento\Checkout\Block\Checkout\LayoutProcessorInterface[]
+     * @var array|LayoutProcessorInterface[]
      */
     protected $layoutProcessors;
 
     /**
-     * @var \Magento\Framework\Serialize\Serializer\Json
+     * @var Json
      */
     private $serializer;
 
     /**
-     * @param \Magento\Framework\View\Element\Template\Context $context
-     * @param \Magento\Customer\Model\Session $customerSession
-     * @param \Magento\Checkout\Model\Session $checkoutSession
-     * @param \Magento\Checkout\Model\CompositeConfigProvider $configProvider
+     * @var JsonHexTag
+     */
+    private $jsonHexTagSerializer;
+
+    /**
+     * @param Context $context
+     * @param customerSession $customerSession
+     * @param checkoutSession $checkoutSession
+     * @param CompositeConfigProvider $configProvider
      * @param array $layoutProcessors
      * @param array $data
-     * @param \Magento\Framework\Serialize\Serializer\Json|null $serializer
+     * @param Json|null $serializer
+     * @param JsonHexTag|null $jsonHexTagSerializer
      * @throws \RuntimeException
      */
     public function __construct(
-        \Magento\Framework\View\Element\Template\Context $context,
-        \Magento\Customer\Model\Session $customerSession,
-        \Magento\Checkout\Model\Session $checkoutSession,
-        \Magento\Checkout\Model\CompositeConfigProvider $configProvider,
+        Context $context,
+        customerSession $customerSession,
+        checkoutSession $checkoutSession,
```

Comment:
Class names and aliases should be PascalCase

Reply:
sure, I will update

Reply:
@lbajsarowicz I have updated the changes. please review now.